cmd/litcli/autopilot.go

```diff
@@ -65,6 +66,17 @@ var addAutopilotSessionCmd = cli.Command{
 				"perform actions on. In the " +
 				"form of: peerID1,peerID2,...",
 		},
+		cli.StringSliceFlag{
+			Name: "configuration",
+			Usage: `If set, this flag must be repeated for each ` +
+				`feature (corresponding to the feature ` +
```

Comment:
Hmm I find this a little strange, and think this will be a bit unintuitive for the user. I think it will be hard for users to understand that they should: 
1. Repeat the configuration flag for every feature in the order they want to specify the config for.
2. If they have set a config for ANY feature, they must set the config to `{}` for every other feature they don't want to configure.

In my opinion I think it would make more sense to the user to specify this as a json object IF they have more than one feature added to the `autopilot a` cmd.

If they have multiple features in the `autopilot a` cmd, they have to specify which feature a config is for as a json header:
Ie. if they feature_1 & feature_2 in the request it would be:

`configuration={"feature_1":{"version":0,"option1":"parameter1","option2":"parameter2",...}, "feature_2":{"version":0,"option1":"parameter1","option2":"parameter2",...}}`

Or if they wouldn't want to specify any configuration for feature_2 they'd just leave that one out from the json_object:
`configuration={"feature_1":{"version":0,"option1":"parameter1","option2":"parameter2",...}}`

Optionally, if they have just 1 feature in the command, we could make it ok for them to set:
configuration={"version":0,"option1":"parameter1",`"option2":"parameter2",...}
Though this might add some extra confusion but be more convenient for users. Leave this one out if you think it's too confusing.

Given that the `channel-restrict-list` & the `peer-restrict-list` is applied to every feature of the `autopilot a` cmd, I think users will expect the `configuration` to be as well.

This isn't blocking though, and if you both think the implemented version is better, feel free to ignore this :).
If you choose to ignore this, I think the first words in this description should be changed from:
"If set" to "If set for any feature"

Reply:
just want to leave one note here that we really dont expect anyone but power users to use LiT via Lncli. The vast vast vast majority of users are expected to use LiT via the UI

Reply:
yes, I agree with @ellemouton , that this is more for power users, which is why I kept the repeated config flags. I think it's also more difficult to use with every layer of json nesting that's added, and we would add redundancy with respect to the feature keys (if we would be consequential, we would also need to remove the `--feature` flags). However, I see the need to change the channel and peer restrict lists to repeated flags as they may need to be interpreted per feature, which is a good point :+1:!

Reply:
Ok makes sense!

> I see the need to change the channel and peer restrict lists to repeated flags as they may need to be interpreted per feature, which is a good point 

Thanks for fixing this with the latest push :)